README.md

```diff
@@ -25,8 +25,8 @@ Commands:
   help [<command>...]
     Show help.
 
-  export [<flags>]
-    Export Observability Data into popular analytics formats.
+  export --matchers=MATCHERS --min-time=MIN-TIME --max-time=MAX-TIME --resolution=RESOLUTION --out=OUT [<flags>]
+    Export observability fata into popular analytics formats.
```

Comment:
is `fata` intentional or was it meant to be `data`?

Reply:
nope, typo (: thanks for spotting